modules/master-node-sizing.adoc

```diff
@@ -26,11 +26,23 @@ The control plane node resource requirements depend on the number of nodes in th
 
 |===
 
+On a cluster with three masters or control plane nodes, the CPU and memory usage will spike up when one of the nodes is stopped, rebooted or fails since the remaining two nodes must handle the load in order to be highly available. This is expected during the upgrades as well since the masters are cordoned, drained and rebooted serially to apply the os updates in addition to the control plane operators update. It is recommended to have the overall resource usage on the master nodes at least around half of all available capacity to handle the resource usage spikes in order to avoid cascading failures on large and dense clusters. Increase the CPU and Memory on the Master nodes accordingly.
+
+[IMPORTANT]
+====
+The node sizing varies depending on the number of nodes and object counts in the cluster. It also depends on whether the objects are actively being created on the cluster since the control plane will be more active in terms of resource usage during the objects creation when compared to state where the objects are all in running phase.
+====
+
 [IMPORTANT]
 ====
 Because you cannot modify the control plane node size in a running {product-title} {product-version} cluster, you must estimate your total node count and use the suggested control plane node size during installation.
 ====
 
+[IMPORTANT]
+====
+The recommendations are based on the data points captured on OpenShift clusters with OpenShiftSDN as the network plugin.
```

Comment:
{product-title} clusters with OpenShift SDN as the network plug-in

Reply:
Updated. Thanks.